mail/models.py

```diff
@@ -34,3 +34,20 @@ class AutomaticEmail(TimestampedModel):
     def __str__(self):
         """String representation of AutomaticEmail"""
         return "AutomaticEmail sender={}, subject={}".format(self.sender_name, self.email_subject)
+
+
+class SentAutomaticEmail(TimestampedModel):
+    """
+    Keeps track of automatic emails which were sent to particular users
+    """
+    user = models.ForeignKey(User, null=False, on_delete=models.CASCADE)
+    automatic_email = models.ForeignKey(AutomaticEmail, null=False, on_delete=models.CASCADE)
```

Comment:
Just want to make sure we really want these `on_delete=models.CASCADE` properties. This just means that when a `User` is deleted, we want to delete this too, and not the other way around, right? Part of the reason I ask is that this property is absent in other places where we might expect it, but the deletion behavior seems to work anyway. `CourseRun` for example. The `course` property is defined as `models.ForeignKey(Course, null=True)`. Are we just setting this explicitly here because it's a Django best practice?